local/network.go

```diff
@@ -367,7 +397,7 @@ func NewDefaultConfigNNodes(binaryPath string, numNodes uint32, networkID uint32
 		nodeConfigs = append(nodeConfigs, nodeConfig)
 		port += 2
 	}
-	if int(numNodes) == 1 && !utils.IsPublicNetwork(networkID) {
+	if int(numNodes) == 1 && !utils.IsPublicNetwork(networkID) && len(beaconConfig) == 0 {
```

Comment:
maybe we can add a comment here for when a network has sybil protecttion disabled

Reply:
not a part of the PR